src/nvim/path.c

```diff
@@ -773,7 +771,7 @@ static void expand_path_option(char_u *curdir, garray_T *gap)
   char_u      *path_option = *curbuf->b_p_path == NUL
                              ? p_path : curbuf->b_p_path;
   char_u      *buf;
-  int len;
+  size_t len;
```

Comment:
`len` can be declared three times in the three blocks where it's used. This reduces the scope and the number of uninitialized variables.